pkg/pipeline/transform/transform_network.go

```diff
@@ -184,6 +185,40 @@ func fillInK8s(outputEntry config.GenericMap, rule api.NetworkTransformRule) {
 	}
 }
 
+const nodeZoneLabelName = "topology.kubernetes.io/zone"
```

Comment:
Would this be configurable ? I see `topology.kubernetes.io/region` & `topology.kubernetes.io/zone` and I wonder if some clusters have only region for example :thinking: 

https://kubernetes.io/docs/reference/labels-annotations-taints/#topologykubernetesioregion

Reply:
Zone and region are not exactly the same thing, generally a region has multiple zones.

In your example `us-east-1` vs `us-east-1c`

Reply:
so are we sure we always have zones when we have region ?

Reply:
Ok sounds legit. Thanks !